[Hackathon]Move complex tests to test_complex.py

nikithamalgifb opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#55514 [Hackathon]Move complex tests to test_complex.py**

Differential Revision: [D27679881](https://our.internmc.facebook.com/intern/diff/D27679881)

Commit:
Move complex tests to test_complex.py

[ghstack-poisoned]

walterddr reviewed (approved):
lgmt. please fix flake8 linter before landing :-)

Commit:
Update on "[Hackathon]Move complex tests to test_complex.py"




[ghstack-poisoned]

Commit:
Update on "[Hackathon]Move complex tests to test_complex.py"


Differential Revision: [D27679881](https://our.internmc.facebook.com/intern/diff/D27679881)

[ghstack-poisoned]

Commit:
Update on "[Hackathon]Move complex tests to test_complex.py"


Differential Revision: [D27679881](https://our.internmc.facebook.com/intern/diff/D27679881)

[ghstack-poisoned]

Commit:
Update on "[Hackathon]Move complex tests to test_complex.py"


Differential Revision: [D27679881](https://our.internmc.facebook.com/intern/diff/D27679881)

[ghstack-poisoned]
